tools/pioasm/c_sdk_output.cpp

```diff
@@ -79,6 +79,13 @@ struct c_sdk_output : public output_format {
         fprintf(out, "#endif\n");
         fprintf(out, "\n");
 
+        fprintf(out, "#ifdef __cplusplus\n");
```

Comment:
https://en.cppreference.com/w/cpp/language/constexpr.html says that `constexpr` was only added in C++11, so should this line perhaps be
```suggestion
        fprintf(out, "#if defined(__cplusplus) && defined(__cpp_constexpr)\n");
```
so that it doesn't break older C++ compilers? (Or perhaps simply `#ifdef __cpp_constexpr` is sufficient?)

Reply:
Yes, fair point.  I've update the check.